services/user.py

```diff
@@ -0,0 +1,63 @@
+from typing import Optional
+
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None,
+) -> User:
+    user = get_user_model().objects.create_user(
+        username=username,
+        password=password,
+    )
+
+    if email:
+        user.email = email
+
+    if first_name:
+        user.first_name = first_name
+
+    if last_name:
+        user.last_name = last_name
+
+    user.save()
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(pk=user_id)
```

Comment:
The `get()` method is used without handling the `DoesNotExist` exception. Consider using `get_object_or_404()` or wrapping this call in a try-except block to handle the exception and avoid runtime errors.